janitor-test/sa.tf

```diff
@@ -0,0 +1,24 @@
+locals {
+  folder_roles = [
+    "roles/resourcemanager.folderAdmin",
+    "roles/resourcemanager.projectCreator",
+    "roles/resourcemanager.projectDeleter",
```

Comment:
does it need projectDeleter if the janitor SA does the deletion?

Reply:
We need to test cleanup already deleted resources. And the easiest ways is to create -> delete -> publish a message. 
That is what we did for other similar tests.

Reply:
ah, good point, thanks!